projects/kit/components/data-list-wrapper/data-list-wrapper-options.ts

```diff
@@ -0,0 +1,26 @@
+import {InjectionToken, ValueProvider} from '@angular/core';
+import {TUI_DEFAULT_STRINGIFY, TuiStringHandler} from '@taiga-ui/cdk';
+import {TuiSizeL, TuiSizeXS} from '@taiga-ui/core';
+
+export interface DataListWrapperOptions<T> {
+    readonly size: TuiSizeXS | TuiSizeL;
+    readonly stringify: TuiStringHandler<T>;
+}
+
+export const TUI_DATA_LIST_WRAPPER_DEFAULT_OPTIONS: DataListWrapperOptions<any> = {
+    size: 'm',
+    stringify: TUI_DEFAULT_STRINGIFY,
+};
+
+export const TUI_DATA_LIST_WRAPPER_OPTIONS = new InjectionToken<
+    DataListWrapperOptions<any>
+>('Default parameters for data list wrapper component', {
+    factory: () => TUI_DATA_LIST_WRAPPER_DEFAULT_OPTIONS,
+});
+
+export const tuiDataListWrapperOptionsProvider: <T>(
+    options: Partial<DataListWrapperOptions<T>>,
+) => ValueProvider = <T>(options: Partial<DataListWrapperOptions<T>>) => ({
+    provide: TUI_DATA_LIST_WRAPPER_OPTIONS,
+    useValue: Object.assign({}, TUI_DATA_LIST_WRAPPER_DEFAULT_OPTIONS, options),
```

Comment:
`{ ...TUI_DATA_LIST_WRAPPER_DEFAULT_OPTIONS, ...options }`

Reply:
`Object.assign` pattern is more efficient than spread

Reply:
Sure, it's not the app state, it's just a bunch of properties but scaled to the size of the UI Kit this could improve performance. Not significantly but still

Reply:
Spread seems to be twice as fast on latest Windows Chrome:
https://jsbench.me/gdkzffgdhb/1

So let's stick to spread and not break the code style.